src/graphics/program-lib/programs/standard.js

```diff
@@ -1141,12 +1144,8 @@ var standard = {
                 } else {
                     code += chunks.normalMapFastPS.replace(/\$UV/g, transformedNormalMapUv);
                 }
-            } else {
+            } else if (!(options.enableGGXSpecular && !options.heightMap)) {
```

Comment:
would this be cleaner:
`} else if (!options.enableGGXSpecular || options.heightMap) {`

Reply:
I did it like this to mirror the logic in a piece of code higher up in the huge generator function (which was the cause of the bug) - I think I prefer to keep it lined up?

Reply:
makes sense.